### PR TITLE
Add activity indicator to create account button.

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-14413
affects: Parent
release note: none

Test plan
1. enable parent qr code flag, enable student qr code
2. scan student qr code so it opens parent app create account screen
3. when you click on the `create account` button you should see an activity indicator.

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/zt3wxha) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fzt3wxha)
teacher: [Link](https://rebrand.ly/em7bb7l) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fem7bb7l)
parent: [Link](https://rebrand.ly/kahksyn) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fkahksyn)

File: Core/Core/Views/ActivityIndicatorButton.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    public required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+
+    func commonInit() {
+        addSubview(spinner)
+        spinner.pinToTopAndBottomOfSuperview()
+        spinner.widthAnchor.constraint(equalToConstant: 30).isActive = true
+        spinner.centerXAnchor.constraint(equalToSystemSpacingAfter: centerXAnchor, multiplier: 1).isActive = true
```
Comment (vanwagonet) [inline Core/Core/Views/ActivityIndicatorButton.swift:38]: why not `constraint(equalTo: centerXAnchor)`?

File: Core/Core/Views/ActivityIndicatorButton.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        spinner.isHidden = true
+        spinner.tintColor = .named(.backgroundLightest)
```
Comment (vanwagonet) [inline Core/Core/Views/ActivityIndicatorButton.swift:40]: `tintColor` (inherited) gets used if `color` isn't set. Why set this right before setting color?

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-14413](https://instructure.atlassian.net/browse/MBL-14413)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.65% | 93.66% | -0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**